### DISCUSSION TITLE
New cable Type

### DISCUSSION THREAD

**Discussion by bclanton40** [netbox-community/netbox#11329] [General]
I want to create a cable type..."Thin cable" since we are switching out our switch cables to 'thin' for better cable management but leaving some of the inter-rack and 'VOIP' cables with the standard cat 5/6 cable.

I went into the cable.py code but it appears that variables are set that might be referenced elsewhere.  Is it possible to accomplish this by making an entry in one location?

**Comment by zmaster7** [comment]
I'm not so sure that it would be classified as a different cable "type," since they are still Cat6/6a, etc cables. There seems to be conflicting naming of these cable options as well, such as "thin" or "slim." Personally, I would classify these with a new field for wire gauge, but not sure if it would be worth the hassle of adding into NetBox. Other options include creating a custom field with a drop down for "Standard" or "Thin" cables (defaulting to standard).
